### ISSUE TITLE
LuLu requires macOS 10.12+

### ISSUE THREAD

**Issue by gondo** [objective-see/LuLu#31]
any reason why 10.11.* is not supported?

**Comment by snappieT** [comment]
Experiencing this too. Surprisingly, the release notes for [0.9.2](https://github.com/objective-see/LuLu/releases/tag/0.9.2) say "support to older versions (OS X 10.9+)"

**Comment by mischievous** [comment]
The code is using at least one function/message in the NSRunloop that showed up in 10.12.

**Comment by mischievous** [comment]
Nope... Something is also using the ColorSync framework which showed up on 10.13.

**Comment by objective-see** [comment]
I've updated the installer checks, build target(s), and documentation to be consistent.  @mischievous i've
